src/pybind/dlpack/dlpack_pybind.cc

```diff
@@ -192,18 +168,15 @@ py::capsule MatrixToDLPack(py::object obj) {
   return py::object(capsule, is_borrowed);
 }
 
-py::capsule CuVectorToDLPack(py::object obj) {
-#if HAVE_CUDA == 1
-  auto* v = obj.cast<CuVector<float>*>();
-  auto* managed_tensor =
-      CreateDLManagedtensor(kDLGPU, CuDevice::GetCurrentDeviceId(), v->Data());
-
+template <typename V>
+py::capsule VectorToDLPackImpl(py::object obj, DLManagedTensor* managed_tensor,
```

Comment:
Is it necessary to have this order of arguments?  The google style guide requires output arguments to be last, so unless there is a good reason, it would be better to have `managed_tensor` be the last arg.  (And I'm not sure how `obj` fits into all of this).

Reply:
I agree.

I will change it now.

Reply:
Now it is changed to pass `py::object` by pointer and `DLManagedTensor*` is the last parameter.